batch_inference_with_ts.html

```diff
@@ -476,10 +436,10 @@ <h2>Conclusion<a class="headerlink" href="#id5" title="Permalink to this headlin
   
     <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
       
-        <a href="custom_service.html" class="btn btn-neutral float-right" title="Custom Service" accesskey="n" rel="next">Next <img src="_static/images/chevron-right-orange.svg" class="next-page"></a>
+        <a href="custom_service.html" class="btn btn-neutral float-right" title="Contents of this Document" accesskey="n" rel="next">Next <img src="_static/images/chevron-right-orange.svg" class="next-page"></a>
```

Comment:
@jlin27 Will the title read a "Contents of this Document" on the pages? This is repeating on almost all the pages

Reply:
Yes will show up. Here is a preview of that page: https://jlin27.github.io/serve-1/batch_inference_with_ts.html

Currently the website also contains this header: https://pytorch.org/serve/batch_inference_with_ts.html

That's because the original document contains that header: https://github.com/pytorch/serve/blob/master/docs/batch_inference_with_ts.md